src/Infrastructure/RestApi/TranslatedDegreeProgramController.php

```diff
@@ -176,6 +183,38 @@ public function get_items($request): WP_Error|WP_REST_Response
         return $response;
     }
 
+    public function getIndex(WP_REST_Request $request): WP_Error|WP_REST_Response
+    {
+        $criteria = CollectionCriteria::new()
+            ->withoutPagination();
+
+        $views = $this->degreeProgramCollectionRepository->findTranslatedCollection(
+            $criteria,
+            MultilingualString::DE
```

Comment:
Although this is not defined by the client, does it make sense to have support for a `lang` query param. I guess we already have it here https://github.com/RRZE-Webteam/FAU-Studium-Common/blob/381097c700ebd3c42f4761dd215c8a9f2b641f4f/src/Infrastructure/RestApi/TranslatedDegreeProgramController.php#L122